frontend/components/commons/sidebar/SidebarMenu.vue

```diff
@@ -118,7 +122,11 @@ $sidebar-button-size: 45px;
   }
   &__info {
     position: relative;
-    margin-bottom: 2em;
+    margin-bottom: 1em;
+    &.--ungruped {
+      margin-top: auto;
+      margin-bottom: 0em;
+    }
   }
```

Comment:
if it's possible easily, replace all the margin by a gap. If not, centralized the margin in the parent selector. This help to not have too much custom space in nested elements